doc/source/whatsnew/v2.1.0.rst

```diff
@@ -21,8 +21,47 @@ enhancement1
 
 .. _whatsnew_210.enhancements.enhancement2:
 
-enhancement2
-^^^^^^^^^^^^
+``map(func, na_action="ignore")`` now works for all array types
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When given a callable, :meth:`Series.map` applies the callable to all elements of the ``Series``.
+Similarly, :meth:`DataFrame.applymap` applies the callable to all elements of the ``DataFrame``,
+while :meth:`Index.map` applies the callable to all elements of the ``Index``.
+
+Frequently, it is not desirable to apply the callable to nan-like values of the array and to avoid doing
+that, the ``map`` method could be called with ``na_action="ignore"``, e.g. like this: ``ser.map(func, na_action="ignore")``.
+However, ``na_action="ignore"`` was not implemented for many ``ExtensionArray`` and ``Index`` types
```

Comment:
Could you note specifically which types `na_action` is now consistently respected? (the datetime likes and period)?

Also could you describe the deprecation in the `na_action` default  here too?